lib/android-helpers.js

```diff
@@ -335,6 +335,14 @@ helpers.initUnicodeKeyboard = async function (adb) {
   return defaultIME;
 };
 
+helpers.setMockLocationApp = async function (adb, app) {
+  if (adb.getApiLevel() < 23) {
```

Comment:
i believe `getApiLevel` is an `async` function, so this should be:

```js
if (await adb.getApiLevel() < 23) {
```

this was probably "working" because `Object < 23` evaluates to `false`, so you fall into the "correct" conditional fork.